### ISSUE TITLE
No access to MFCAuto

### ISSUE THREAD

**Issue by horacio9a** [sstativa/mfc-node#28]
Several times during this year I tried to install MFCAuto but I always get almost the same error like this one:

650 error Windows_NT 6.1.7601
651 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Horacio\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install"
652 error node v8.1.3
653 error npm  v4.1.2
654 error code 128
655 error Command failed: git -c core.longpaths=true clone --template=C:\Users\Horacio\AppData\Roaming\npm-cache\_git-remotes\_templates --mirror git@github.com:zombiealex/MFCAuto.git C:\Users\Horacio\AppData\Roaming\npm-cache\_git-remotes\git-github-com-zombiealex-MFCAuto-git-fa04cc97
655 error Cloning into bare repository 'C:\Users\Horacio\AppData\Roaming\npm-cache\_git-remotes\git-github-com-zombiealex-MFCAuto-git-fa04cc97'...
655 error Host key verification failed.
655 error fatal: Could not read from remote repository.

npm-debug.log is longer but I think that's enough.
What should I do?

**Comment by sstativa** [comment]
I don't know, I'm not a Windows user.

I would recommend to 
* try to reinstall [git](https://git-scm.com/download/win)
* try to clone MFCAuto independently via ssh
  `git clone git@github.com:ZombieAlex/MFCAuto.git`

If the last one did not work to you, you probably would have to create private and public key for your  pc.

**Comment by marabu88** [comment]
I am windows user and don't have any problems with install MFCAuto. I just use command (without admin rights) from folder with script
`npm install`
This command loads all the necessary modules, including MFCAuto

**Comment by horacio9a** [comment]
Thank you very much @sstativa and @marabu88 for help.
The problem I solved with reinstalling git but this time I chose different options than the first time. 
There are 5 options and I've taken screenshot for all but I guess maybe this last one was important.
And for the first time I am used the MINGW32 window rather than the Command Prompt window as it used before.
![2331](https://user-images.githubusercontent.com/17076459/28010336-e300cc3e-655e-11e7-90b1-18b664699c71.jpg)
![2332](https://user-images.githubusercontent.com/17076459/28010337-e30c7e8a-655e-11e7-9a36-0de1da5bee32.jpg)
